storage/src/backend/rocksdb_backend.rs

```diff
@@ -43,101 +29,53 @@ pub struct RocksDBBackendStats {
-            .map(|_| true)
+impl GarbageCollector for RocksDBBackend {
+    fn new_cycle_started(&mut self) -> Result<(), StorageBackendError> {
+        Ok(())
     }
 
-    fn put_batch(
+    fn mark_reused(
         &mut self,
-        batch: Vec<(EntryHash, ContextValue)>,
+        _reused_keys: std::collections::HashSet<EntryHash>,
     ) -> Result<(), StorageBackendError> {
-        let mut rocksb_batch = WriteBatch::default(); // batch containing DB key values to persist
+        Ok(())
+    }
+}
+
+impl KeyValueStoreBackend<MerkleStorage> for RocksDBBackend {
+    fn is_persistent(&self) -> bool {
+        (self.inner.deref() as &dyn KeyValueStoreBackend<MerkleStorage>).is_persistent()
+    }
 
-        for (k, v) in batch.iter() {
-            (self.inner.deref() as &dyn KeyValueStoreWithSchema<MerkleStorage>).put_batch(
-                &mut rocksb_batch,
-                &k,
-                &v,
-            )?;
-        }
+    fn put(&self, key: &EntryHash, value: &ContextValue) -> Result<(), DBError> {
+        (self.inner.deref() as &dyn KeyValueStoreBackend<MerkleStorage>).put(key, value)
+    }
 
-        (self.inner.deref() as &dyn KeyValueStoreWithSchema<MerkleStorage>)
-            .write_batch(rocksb_batch)?;
+    fn delete(&self, key: &EntryHash) -> Result<(), DBError> {
+        (self.inner.deref() as &dyn KeyValueStoreBackend<MerkleStorage>).delete(key)
+    }
 
-        Ok(())
+    fn merge(&self, key: &EntryHash, value: &ContextValue) -> Result<(), DBError> {
+        (self.inner.deref() as &dyn KeyValueStoreBackend<MerkleStorage>).merge(key, value)
     }
 
-    fn merge(&mut self, key: &EntryHash, value: ContextValue) -> Result<(), StorageBackendError> {
-        let cf = self.inner.cf_handle(&self.column_name).ok_or(
-            StorageBackendError::MissingColumnFamily {
-                name: &self.column_name,
-            },
-        )?;
-
-        self.inner
-            .merge_cf_opt(
-                cf,
-                &key.as_ref()[..],
-                &value,
-                &Self::default_write_options(),
-            )
-            .map_err(StorageBackendError::from)
+    fn get(&self, key: &EntryHash) -> Result<Option<ContextValue>, DBError> {
+        (self.inner.deref() as &dyn KeyValueStoreBackend<MerkleStorage>).get(key)
     }
 
-    /// Warning: always returns None.
-    fn delete(&mut self, key: &EntryHash) -> Result<Option<ContextValue>, StorageBackendError> {
-        let cf = self.inner.cf_handle(&self.column_name).ok_or(
-            StorageBackendError::MissingColumnFamily {
-                name: &self.column_name,
-            },
-        )?;
-
-        self.inner
-            .delete_cf_opt(cf, &key.as_ref()[..], &Self::default_write_options())
-            .map_err(StorageBackendError::from)?;
-        Ok(None)
+    fn contains(&self, key: &EntryHash) -> Result<bool, DBError> {
+        (self.inner.deref() as &dyn KeyValueStoreBackend<MerkleStorage>).contains(key)
```

Comment:
@mat13mn 
I would wrap this to some fn or macro, something like that, if possible
```
fn merkle() ->&dyn KeyValueStoreBackend<MerkleStorage> {
  self.inner.deref() as &dyn KeyValueStoreBackend<MerkleStorage>
}
```